PrepPy/train_valid_test_split.py

```diff
@@ -121,4 +119,12 @@ def train_valid_test_split(X, y, valid_size=0.25, test_size=None, stratify=None,
                                                            random_state=random_state,
                                                            shuffle=shuffle)
                                                            
-    return X_train, X_valid, X_test, y_train, y_valid, y_test
+    splits = {'X_train': pd.DataFrame(X_train),
+              'X_valid': pd.DataFrame(X_valid), 
+              'X_test': pd.DataFrame(X_test),
+              'y_train': pd.DataFrame(y_train),
+              'y_valid': pd.DataFrame(y_valid),
+              'y_test': pd.DataFrame(y_test)}
+            
+        
+    return list(splits.values())
```

Comment:
I think your `splits` dictionary makes more sense as an output than a list. For example, if users want to get `X_train` they can just type `output['X_train']` instead of `output[0]`. 

```suggestion
    return splits
```

Reply:
@jasmineqyj  Good suggestion, thanks